src/mobile-token/utils.ts

```diff
@@ -78,17 +94,20 @@ export const getMobileTokenErrorHandlingStrategy = (
     errorResolution = err.resolution;
   } else if (err instanceof RemoteTokenStateError) {
     errorResolution = mapTokenErrorResolution(err);
-  } else if (isEntityDeletedError(err)) {
-    /**
-     * This handles the situation where local token is still stored, while
-     * the remote token is already deleted, should only happens with a very
-     * old account that got their account history truncated in backoffice.
-     */
-    return 'reset';
   } else {
     return 'unspecified';
   }
 
+  logToBugsnag(
+    `mobile token handle error ${err} with resolution ${errorResolution}`,
+  );
+
+  notifyBugsnag(err, {
+    errorGroupHash: 'token',
+    metadata: {
+      description: `Handling error ${err} with resolution of ${errorResolution}`,
+    },
+  });
```

Comment:
Do we need to do both log and notify right after each other?

Reply:
But we don't need a breadcrumb when we right after generate an error for the same thing?

Reply:
In my understanding, if we generate error first, the breadcrumb that gets called next won't get sent. I added this to at least get more logs in Bugsnag, since we need to identify the exact point of errors on the token, and the current bugsnag logs doesn't seem to give us a clear image of what actually happens.